### PR TITLE
[APPWIZ] Release download_binding from client after abort

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: [CORE-15786](https://jira.reactos.org/browse/CORE-15786)

## Proposed changes

- Don't release binding `OnStopBinding`.
- After abort, release `download_binding` set `NULL`.

## TODO

- [ ] Do tests.

### CODE REVIEW

**Comment by katahiromz** [comment]
CoInitializeEx

File: dll/cpl/appwiz/addons.c [hunk 1]
```diff
@@ -411,12 +404,25 @@ static INT_PTR CALLBACK installer_proc(HWND hwnd, UINT msg, WPARAM wParam, LPARA
...
-            if(download_binding) {
+            if (download_binding)
+            {
                 IBinding_Abort(download_binding);
+
+                /*
+                 * https://learn.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/ms775068(v=vs.85)
+                 *
+                 * > After it aborts the bind operation, the client must release any pointers that
+                 * > were obtained during the binding.
```
Comment (HBelusca) [inline dll/cpl/appwiz/addons.c:417]:
```suggestion
                 * > After it aborts the bind operation, the client must release
                 * > any pointers that were obtained during the binding.
```

**Review by HBelusca — COMMENTED** [review]
May make sense. Someone else with COM knowledge may be able to confirm.

**Comment by katahiromz** [comment]
If there was any recurrence after #5551, I will plan to merge this PR.
Making this PR "Draft"...

**Comment by katahiromz** [comment]
Closed.

**Pull request closed without merging**